common/src/main/java/com/wynntils/models/character/CharacterModel.java

```diff
@@ -125,6 +125,10 @@ public boolean isVeteran() {
         return isVeteran;
     }
 
+    public boolean isHunted() {
```

Comment:
```suggestion
    public boolean isHuntedMode() {
```

Or something else, I think this name needs to be a bit better.